### ISSUE TITLE
Enable Amazon.com gift card gifting via email

### ISSUE THREAD

**Issue by jameswalpole** [bitpay/copay#5235]
I want to be able to send Amazon gift cards I bought with bitcoin to my friends. Can this be done in the BitPay app?

**Comment by gabrielbazan7** [comment]
You can not share directly from the app. But you can copy to clipboard or take a screenshot of the Claim Code and share that to your friends.

**Comment by jameswalpole** [comment]
Yup, but it would be pretty awesome if you could do this directly from the app, or if we at least indicated that it's possible to copy and send the code to someone else as a gift. Would also help spread the BitPay app.
